### PR TITLE
CB-1463 FreeIPA implement POST create request

### PR DESCRIPTION
**Pull request by lacikaaa**
- added Credential to request until Environment service is ready
- response will be done in another PR (GET implementation)

### CODE REVIEW

File: freeipa/src/main/java/com/sequenceiq/freeipa/converter/FreeIpaRequestToFreeIpaConverter.java [hunk 1]
```diff
@@ -3,13 +3,13 @@
 import org.springframework.core.convert.converter.Converter;
 import org.springframework.stereotype.Component;
 
-import com.sequenceiq.freeipa.api.model.freeipa.FreeIpaRequest;
+import com.sequenceiq.freeipa.api.v1.freeipa.stack.model.FreeIpaServerRequest;
 import com.sequenceiq.freeipa.entity.FreeIpa;
 
 @Component
-public class FreeIpaRequestToFreeIpaConverter implements Converter<FreeIpaRequest, FreeIpa> {
+public class FreeIpaRequestToFreeIpaConverter implements Converter<FreeIpaServerRequest, FreeIpa> {
```
Comment (handavid) [inline freeipa/src/main/java/com/sequenceiq/freeipa/converter/FreeIpaRequestToFreeIpaConverter.java]: rename to FreeIpaServerRequestToFreeIpaConverter to match conversion classes

File: freeipa/src/main/java/com/sequenceiq/freeipa/converter/credential/CredentialV4RequestToCredentialConverter.java [hunk 2]
```diff
@@ -16,7 +17,7 @@
     @Override
```
Comment (handavid) [inline freeipa/src/main/java/com/sequenceiq/freeipa/converter/credential/CredentialV4RequestToCredentialConverter.java]: This probably doesn't convert CredentialV4Requests anymore after all the refactoring. Rename to CredentialRequestToCredentialConverter.

I also noticed that some other converters no longer implement Converter<?,?>. e.g., InstanceTemplateRequestToTemplateConverter and InstanceGroupRequestToInstanceGroupConverter. Should they all implement Converter or not?

Comment (lacikaaa) [reply freeipa/src/main/java/com/sequenceiq/freeipa/converter/credential/CredentialV4RequestToCredentialConverter.java]: Converter supports only one parameter, and I want to use more. As I don't think we want to use ConversionService to get converters, implementing the interface is not necessary

File: freeipa/src/main/java/com/sequenceiq/freeipa/converter/FreeIpaRequestToFreeIpaConverter.java (hunk 1 above)
Comment (lacikaaa) [reply freeipa/src/main/java/com/sequenceiq/freeipa/converter/FreeIpaRequestToFreeIpaConverter.java]: fixed these

File: freeipa/src/main/java/com/sequenceiq/freeipa/converter/credential/CredentialV4RequestToCredentialConverter.java (hunk 2 above)
Comment (handavid) [reply freeipa/src/main/java/com/sequenceiq/freeipa/converter/credential/CredentialV4RequestToCredentialConverter.java]: thanks for the info!

**Pull request merged**